File: 3-python/s4_ifs.py
nota=float(input("Ingrese nota:"))
if nota<0 or nota > 20:
    print("Nota invalida.")
else:
    if nota>17:
        print("Excelente.")
    elif nota>11:
        print("Aprobado.")
    else:
        print("Desaprobado.")

File: 2-c#/s4_ifs.cs
Console.WriteLine("Ingrese nota:");
double nota= double.Parse(Console.ReadLine());
if (nota<0 || nota>20){
    Console.WriteLine("Nota no valida, solo rango de 0 a 20");
}else{
    if (nota >= 17){
        Console.WriteLine("Excelente");
    }else if(nota >= 11){
        Console.WriteLine("Aprobado");
    }else{
        Console.WriteLine("Desaprobado");
    }
}